sklearn/preprocessing/tests/test_target_encoder.py

```diff
@@ -187,18 +186,19 @@ def test_errors(y, msg):
         enc.fit_transform(X, y)
 
 
-def test_use_regression_target():
+def test_use_regression_target(global_random_seed):
```

Comment:
I do not think this is worth using `global_random_seed` here because this test is only checking that the target is inferred properly.